### ISSUE TITLE
Fangfang Wan

### ISSUE THREAD

**Issue by bensoltoff** [uchicago-computation-workshop/2019_spring_conf#6]

**Comment by bensoltoff** [comment]
* Interesting design to measure economic growth. Why not traditional measures?
* Scatterplot is really hard to follow. Colors do not distinguish between province. Better to use a line chart connecting each data point for each province

**Comment by rickecon** [comment]
* Good definition of fiscal centralization
* Is the nightlight data the computational component of your thesis? Did you do anything computational or did you just take that data from some source and use it in your regressions? If the latter is true, that is probably not computational enough.

**Comment by Fangfangwan** [comment]
> * Interesting design to measure economic growth. Why not traditional measures?
> * Scatterplot is really hard to follow. Colors do not distinguish between province. Better to use a line chart connecting each data point for each province

I also used traditional measurement for economic growth, and I'm comparing them.
Some studies posed doubts on traditional economic growth measures (Martinez, 2017), and they have designed various methods to revise GDP measurement using nightlight data (Henderson et al., 2012).

References: 
Henderson, J. V., Storeygard, A., & Weil, D. N. (2012). Measuring economic growth from outer space. American economic review, 102(2), 994-1028.
Martinez, L. R. (2018). How Much Should We Trust the Dictator's GDP Estimates?. Available at SSRN 3093296.

**Comment by Fangfangwan** [comment]
> * Good definition of fiscal centralization
> * Is the nightlight data the computational component of your thesis? Did you do anything computational or did you just take that data from some source and use it in your regressions? If the latter is true, that is probably not computational enough.

I downloaded raw data from NOAA, but they are presented as total nighlight amount in a certain area during a year, and they are presented at county level. I normalized them using total population as weights, and converted them to provincial level data.  Is that computational enough? Also, I used both Python (machine learning methods for OLS & plotting) and Stata in my thesis, and there are nearly 50 regressions included in my thesis. Do they count towards the computational part?

**Comment by Fangfangwan** [comment]
@bensoltoff @rickecon I'm not sure if you could receive notifications on my reply of your questions or not, so here they are!

**Comment by yilundai** [comment]
Very thought-provoking topic! I wish my income tax rate could be lower!
